samples/join-team-using-qr-code/nodejs/Readme.md

```diff
@@ -0,0 +1,105 @@
+---
+page_type: sample
+description: This sample demos joining a team using QR code containing the team id.
```

Comment:
This sample demos a feature where you can join a team using QR code containing the team's id.